jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/TabsAssert.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.jdi.light.angular.asserts;
+
+import com.epam.jdi.light.angular.elements.complex.Tabs;
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import org.hamcrest.Matchers;
+
+import java.util.List;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+public class TabsAssert extends UIAssert<TabsAssert, Tabs> {
+
+    @JDIAction("Assert that '{name}' has tabs count")
```

Comment:
Why not check the input parameter "tabsCountForTest" here?
For example,
@JDIAction("Assert that '{name}' has '{0}' tabs")